etc/adminhtml/system.xml

```diff
@@ -32,6 +32,10 @@
                     <label>Sandbox Public Key</label>
                     <comment><![CDATA[Remember to <a href="//pagseguro.ricardomartins.net.br/sandbox.html" target="_blank">authorize your sandbox</a> account before using this feature.]]></comment>
                     <depends><field id="sandbox">1</field></depends>
+                </field>
+                <field id="creditcard_ux" translate="label comment" type="select" sortOrder="3" showInDefault="1" showInWebsite="1" showInStore="0">
```

Comment:
This is a config related to credit card only. This way, this must go under the "Credit Card" group, not in the general settings of the module.